src/redux/actions/searchMovies.js

```diff
@@ -0,0 +1,5 @@
+export const searchMovies = text => ({
+  type: "SEARCH_MOVIES",

```

Comment:
`"SEARCH_MOVIES"` string are repeated 3 times. Common practice is to create a variabled for action name and than use it everywhere. Somelithing like this:

```
export const SEARCH_MOVIES = 'SEARCH_MOVIES'
... // other action names

---

import { SEARCH_MOVIES } from './path-to-actions'

export const searchMovies = text => ({
  type: SEARCH_MOVIES,
  payload: text
})
```